std/typecons.d

```diff
@@ -5189,6 +5189,8 @@ mixin template Proxy(alias a)
                 return a.opCmp(b);
             else static if (is(typeof(b.opCmp(a))))
                 return -b.opCmp(a);
+            else static if (isFloatingPoint!ValueType || isFloatingPoint!B)
+                return a < b ? -1 : a > b ? +1 : a == b ? 0 : float.nan;
```

Comment:
What matters is what happens when it's inlined in an actual comparison, because if this function isn't being inlined then a few extra instructions will be the least of your worries.